### PR TITLE
Remove 'excessive parentheses' linter warnings

### PR DESCRIPTION
**Pull request by TemporalOroboros**
## About the PR

This disables the VSCode linter warnings for "excessive parentheses"
The relevant linter warnings were set to never allow non arithmetically-necessary parentheses which resulted in it wanting to strip the first set of parentheses in `(foo * bar) / (foo + bar)` ( -> `foo * bar / (foo + bar)`.
This tends to make code slightly less clear so it should go.

**Media**

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -30,9 +30,9 @@ dotnet_style_predefined_type_for_locals_parameters_members = true:suggestion
 dotnet_style_predefined_type_for_member_access = true:suggestion
 
 # Parentheses preferences
-dotnet_style_parentheses_in_arithmetic_binary_operators = never_if_unnecessary:suggestion
-dotnet_style_parentheses_in_other_binary_operators = never_if_unnecessary:suggestion
-dotnet_style_parentheses_in_relational_binary_operators = never_if_unnecessary:suggestion
+#dotnet_style_parentheses_in_arithmetic_binary_operators = never_if_unnecessary:suggestion
+#dotnet_style_parentheses_in_other_binary_operators = never_if_unnecessary:suggestion
+#dotnet_style_parentheses_in_relational_binary_operators = never_if_unnecessary:suggestion
```
Comment (metalgearsloth) [inline .editorconfig:36]: Just delete it.

**Comment by ElectroJr** [comment]
> This tends to make code slightly less clear so it should go.

I'm of the exact opposite opinion: unnecessary parenthesis make code less clear, so IMO the suggestion should stay.

**Comment by wrexbe** [comment]
It's not really one way or the other, it just depends. With less used operators, the order of operations can get confusing. For common ones it's not that helpful.

**Comment by metalgearsloth** [comment]
I am indifferent but someone needs to choose which funny button to press.

**Pull request closed without merging**